.evergreen/functions.yml

```diff
@@ -651,6 +651,22 @@ functions:
 
           npm run --unsafe-perm --workspace compass-e2e-tests test-packaged-ci
 
+  spawn-compass-mongodb-com:
+    - command: git.get_project
+      type: system
+      params:
+        directory: compass-mongodb-com
+    - command: shell.exec
+      type: setup
+      params:
+        working_dir: src
+        shell: bash
+        script: |
+          set -e
+
+          docker build -t compass/web ./
+          docker run -p 8080:8080 -t -e PORT=8080 -e UPDATE_CHECKER_ALLOW_DOWNGRADES=true -d compass/web
```

Comment:
The plan is to run two of these: One that will downgrade the newly packaged compass to the most recent release like this, the other will upgrade from the most recent release to this newly packaged one. This newly packaged version isn't released yet, so we'll have to make some changes to compass-mongodb-com to be able to tell it about the new package.